domain/build.gradle

```diff
@@ -48,6 +48,8 @@ dependencies {
       'androidx.appcompat:appcompat:1.0.2',
       'androidx.lifecycle:lifecycle-livedata-ktx:2.2.0-alpha03',
       'com.google.dagger:dagger:2.24',
+      'com.google.firebase:firebase-analytics-ktx:17.4.2',
+      'com.google.firebase:firebase-crashlytics:17.0.0',
       "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version",
       'androidx.exifinterface:exifinterface:1.0.0-rc01'
```

Comment:
While you are at this PR, please fix this also `'androidx.exifinterface:exifinterface:1.0.0-rc01'`

Reply:
Done.